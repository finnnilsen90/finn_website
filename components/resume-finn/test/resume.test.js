const React = require('react');
const ReactTestUtils = require('react-dom/test-utils');
const renderer = require('react-test-renderer')
let { shallow, mount, render } = require('enzyme');

const Resume = require('../resume-finn.js');
const Outline = require('../lib/outline-resume/outline-resume.js')

let ResumeConst = new Resume(); 

jest.mock('../lib/outline-resume/outline-resume.js', () => 'Outline')

describe('component connection', () => {

    it('should be defined', () => {
        expect(Resume).toBeDefined();
    })

})  
   
describe('<Resume />', () => {

    const wrapper = render(<Resume />);

    it('renders components', () => {
        expect(wrapper).toMatchSnapshot();
    });


})
