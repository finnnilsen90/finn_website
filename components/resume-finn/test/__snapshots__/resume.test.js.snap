// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Resume /> renders components 1`] = `
<div
  class="container"
>
  <div
    class="libcomp-hamburger_Container font"
  >
    <div
      class="ham_container"
    >
      <div
        class="hamburger"
      >
        <hr />
        <hr />
        <hr />
      </div>
    </div>
    <div
      class="dropdown_container"
    >
      <ul
        class="dropdown"
      >
        <li
          class="link ham_link"
        >
          <a
            class="a_link"
            href="/home"
          >
            Home Page
          </a>
        </li>
        <li
          class="link ham_link"
        >
          <a
            class="a_link"
            href="/resume"
          >
            Resume
          </a>
        </li>
        <li
          class="link ham_link"
        >
          <a
            class="a_link"
            href="/project"
          >
            Projects
          </a>
        </li>
        <li
          class="link ham_link"
        >
          <a
            class="a_link"
            href="/contact"
          >
            Contact
          </a>
        </li>
      </ul>
    </div>
  </div>
  <outline />
</div>
`;
